packages/qwik/src/server/api.md

```diff
@@ -74,6 +71,15 @@ export interface PrefetchStrategy {
     symbolsToPrefetch?: SymbolsToPrefetch;
 }
 
+// @public (undocumented)
+export interface PreloaderOptions {
+    activePreloads?: number;

```

Comment:
Hmmm I actually prefer `maxSimultaneousPreloads` although it's a bit verbose. "Simultaneous" is so hard to type. Maybe `maxActivePreloads`, or `maxConcurrentPreloads` would be a little easier?

Reply:
I could accept `maxActivePreloads`, although as the comment says, 100% probable ones have unlimited preloads, so that's a lie, and furthermore the idea is that people hover and read the JSDoc anyway :)

Reply:
Hmmm but this key is for the non-100% probable ones. So maybe `maxBufferedPreloads`?